config/migrate/entry-ids.json

```diff
@@ -36,7 +36,7 @@
         7
     ],
     [
-        "entroy",
+        "entropy",
```

Comment:
I "ctrl+f"ed the entire repo to make sure this config change doesn't break anything.